pkgs/development/interpreters/clojure/obb.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchurl, writeScript }:
+
+stdenv.mkDerivation rec {
+  pname = "obb";
+  version = "0.0.1";
+
+  src = fetchurl {
+    url = "https://github.com/babashka/${pname}/releases/download/v${version}/${pname}.tar.gz";
+    sha256 = "sha256-IgbimykETgUu2ON4OTK6g2oSLvWRnnGV50pJVoiqVk8=";
+  };
+
+  setSourceRoot = "sourceRoot=`pwd`";
+
+  installPhase = ''
+  runHook preInstall
+
+  install -Dm755 obb "$out/bin/obb"
+  '';
+
+  installCheckPhase = ''
+    $out/bin/obb --version | grep '${version}'
```

Comment:
There is [`testVersion`](https://github.com/NixOS/nixpkgs/blob/de49d559382ea05a304589b6fc13c9be68351810/pkgs/build-support/trivial-builders.nix#L724-L755) which can be used here instead. See https://github.com/NixOS/nixpkgs/commit/255c2849248e331d7f0e56a041b80e0014a19177 for an example.


The simple 1 + 2 should also be a [package test](https://nixos.org/manual/nixpkgs/stable/#ssec-inline-package-tests-writing)

Reply:
Version test doesn't even work with 0.0.1, so removing that for now in addition to fixing the package test